scripts/calc_flux_points.py

```diff
@@ -35,4 +48,12 @@ def main(config, dataset_path, best_model_path, output):
 
 
 if __name__ == "__main__":
+    parser = ArgumentParser()
```

Comment:
Why this change? Just curious

Reply:
Needs to be in the main block to import a function from here in a other script.
Otherwise it runs on import and will complain about a mismatch in the arguments.